tests/test_templates_layers.py

```diff
@@ -462,3 +465,69 @@ def test_random_layer_weights(self, n_subsystems, tol):
         params = [q.parameters for q in rec.queue]
         params_flat = [item for p in params for item in p]
         assert np.allclose(weights.flatten(), params_flat, atol=tol)
+
+
+class TestSimplifiedTwoDesign:
+    """Tests for the SimplifiedTwoDesign method from the pennylane.templates.layers module."""
+
+    @pytest.mark.parametrize("n_wires, n_layers, shape_weights", [(1, 2, (0,)),
+                                                                  (2, 2, (2, 1, 2)),
+                                                                  (3, 2, (2, 2, 2)),
+                                                                  (4, 2, (2, 3, 2))])
+    def test_circuit_queue(self, n_wires, n_layers, shape_weights):
+        """Tests the gate types and parameter values in the circuit."""
+        np.random.seed(12)
```

Comment:
Curious about this one :smile:

Reply:
Ah, another one of those copy and paste mistakes. Good find.

Reply:
No wait, this was to initialise the random weights consistently.

Reply:
Oh, you're right! Missed that part here :)